hack/csi/manifests/vsphere/example-vsphere-csi-driver.yaml

```diff
@@ -0,0 +1,692 @@
+apiVersion: storage.k8s.io/v1
```

Comment:
This includes resources that come as part of an openshift install.
Considering this is being framed as: `This commit introduces sample vsphere CSI manifests to be used by the dev and QE`, these resources shouldn't be included.

Reply:
Removed to only include the Windows daemonset